src/sentry/api/event_search.py

```diff
@@ -829,30 +830,47 @@ def to_list(value):
                         project_to_filter = project.id
 
                     kwargs["conditions"].append(converted_filter)
-            elif name == ISSUE_ID_ALIAS and term.value.value != "":
+            elif name == ISSUE_ID_ALIAS and value != "":
                 # A blank term value means that this is a has filter
-                kwargs["group_ids"].extend(to_list(term.value.value))
-            elif name == ISSUE_ALIAS and term.value.value != "":
+                kwargs["group_ids"].extend(to_list(value))
+            elif name == ISSUE_ALIAS and value != "":
                 if params and "organization_id" in params:
                     try:
                         group = Group.objects.by_qualified_short_id(
-                            params["organization_id"], term.value.value
+                            params["organization_id"], value
                         )
                         kwargs["group_ids"].extend(to_list(group.id))
                     except Exception:
                         raise InvalidSearchQuery(
-                            u"Invalid value '{}' for 'issue:' filter".format(term.value.value)
+                            u"Invalid value '{}' for 'issue:' filter".format(value)
                         )
             elif name == USER_ALIAS:
                 # If the key is user, do an OR across all the different possible user fields
                 user_conditions = [
                     convert_search_filter_to_snuba_query(term, key=field)
                     for field in FIELD_ALIASES[USER_ALIAS]["fields"]
                 ]
-                if term.operator == "!=" and term.value.value != "":
+                if term.operator == "!=" and value != "":
                     kwargs["conditions"].extend(user_conditions)
                 else:
                     kwargs["conditions"].append(user_conditions)
+            elif name == RELEASE_ALIAS and params and value == "latest":
+                converted_filter = convert_search_filter_to_snuba_query(
+                    SearchFilter(
+                        term.key,
+                        term.operator,
+                        SearchValue(
+                            parse_release(
+                                value,
+                                params["project_id"],
+                                params.get("environment_objects"),
+                                params["organization_id"],
```

Comment:
@wmak `organization_id` isn't always in params here, and causes this error: https://sentry.io/organizations/sentry/issues/1846529501/?project=1&query=is%3Aunresolved+assigned%3A%23workflow&statsPeriod=14d. Mind taking a look?

Reply:
:+1: can do thanks!

Reply:
Thanks again for the heads up, fix here: https://github.com/getsentry/sentry/pull/20790